Explorer/Assets/DCL/LOD/Systems/UpdateSceneLODInfoSystem.cs

```diff
@@ -72,49 +76,64 @@ private void UpdateLODLevel(ref SceneLODInfo sceneLODInfo, ref PartitionComponen
                 CheckLODLevel(ref partitionComponent, ref sceneLODInfo, sceneDefinitionComponent);
         }
 
+
+        [Query]
+        [None(typeof(DeleteEntityIntention))]
+        private void InstantiateCurrentLOD(ref SceneLODInfo sceneLODInfo, ref SceneDefinitionComponent sceneDefinitionComponent)
+        {
+            var currentLOD = sceneLODInfo.GetCurrentLOD();
+            if (!sceneLODInfo.IsDirty || currentLOD == null) return;
+
+            if (!(frameCapBudget.TrySpendBudget() && memoryBudget.TrySpendBudget())) return;
+
+            if (currentLOD.State == LODAsset.LOD_STATE.WAITING_INSTANTIATION &&
+                currentLOD.AsyncInstantiation.IsWaitingForSceneActivation())
+            {
+                currentLOD.FinalizeInstantiation();
```

Comment:
Why do we need to force `Instantiation` considering we provided the budget? Is not it better to check `AsyncOperation.isDone`?
Otherwise there is less space for maneuver for Unity? WDYT?